### PR TITLE
1436 accept null value to un assign previously added value

### PR DESCRIPTION
**Pull request by swordEdge**
https://www.loom.com/share/1e9f6825db0140b6b61caaf3463935d8

### CODE REVIEW

**Review by six7 — CHANGES_REQUESTED** [review]
I'll test this in a bit, looks good! Only some minor comments

**Comment by six7** [comment]
Test coverage is failing as well, let's get that green, comments resolved and then 👍

File: src/app/components/InspectorTokenSingle.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ import Button from './Button';
 import DownshiftInput from './DownshiftInput';
 import Modal from './Modal';
 import Stack from './Stack';
-import { IconBrokenLink } from '@/icons';
+import { IconBrokenLink, IconValueNone } from '@/icons';
```
[the remark this answers is not in the document]
Comment (swordEdge) [reply src/app/components/InspectorTokenSingle.tsx:19]: > Can you use ValueNoneIcon instead from radix icons?

> Where can I get ValueNoneIcon? Is this already exist in the plugin?

File: src/plugin/asyncMessageHandlers/remapTokens.ts [hunk 2]
```diff
@@ -11,7 +11,7 @@ import { TokenTypes } from '@/constants/TokenTypes';
 export const remapTokens: AsyncMessageChannelHandlers[AsyncMessageTypes.REMAP_TOKENS] = async (msg) => {
   try {
     const {
-      oldName, newName, updateMode, category, tokens,
+      oldName, newName, updateMode, category,
```
[the remark this answers is not in the document]
Comment (swordEdge) [reply src/plugin/asyncMessageHandlers/remapTokens.ts:14]: > Is this change required? What does it get us?

> Not required. Just wanted to remove same name value. There is another `tokens` value in this code.
Anyway revert this change.

File: src/app/components/InspectorTokenSingle.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (six7) [reply src/app/components/InspectorTokenSingle.tsx:19]:
```
import { ValueNoneIcon } from '@radix-ui/react-icons'
```

File: src/plugin/asyncMessageHandlers/setNoneValuesOnNode.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    Object.entries(nodesToRemove).map(async (node) => {
+      const newEntries = node[1].reduce<Record<string, string>>((acc, curr) => {
+        acc[curr] = 'none';
+        return acc;
+      }, {});
+
+      const nodeToUpdate = await defaultNodeManager.getNode(node[0]);
+      if (nodeToUpdate) {
+        await updatePluginData({
+          entries: [nodeToUpdate], values: newEntries, shouldRemove: true, tokensMap,
```
[the remark this answers is not in the document]
Comment (swordEdge) [reply src/plugin/asyncMessageHandlers/setNoneValuesOnNode.ts:29]: I think we're using cache here.

**Review by six7 — APPROVED** [review]

**Review by six7 — APPROVED** [review]

**Pull request merged**